public/js/startup.js

```diff
@@ -40,6 +40,10 @@ pimcore.bundle.web2print.startup = Class.create({
         var menu = e.detail.menu;
         var tree = e.detail.tree;
         var me = this;
+        const perspectiveCfg = pimcore.globalmanager.get("perspective");
```

Comment:
```suggestion
        const perspectiveCfg = e.detail.perspective;
```
and adding `perspective: perspectiveCfg` to
https://github.com/pimcore/admin-ui-classic-bundle/blob/558c7c245208793ed67a413eca86810d75fcb325/public/js/pimcore/document/tree.js#L867

would work